.gitignore

```diff
@@ -1,2 +1,4 @@
 tmp/
 *.png
+.DS_Store
```

Comment:
Let's keep the system and IDE specific stuff in a global gitignore instead of the repo.

Here's one guide showing how to do this: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c.